packages/fxa-auth-server/config/dev.json

```diff
@@ -378,6 +379,17 @@
         "allowedScopes": "https://identity.mozilla.com/account/subscriptions https://identity.mozilla.com/account/newsletters",
         "publicClient": true
       },
+      {
+        "name": "Thunderbird",
+        "id": "a958794159236f76",
+        "hashedSecret": "2e99758548972a8e8822ad47fa1017ff72f06f3ff6a016851f45c398732bc50c",
```

Comment:
I just made these up. Are they supposed to be anything in particular? I don't think the hashed secret is actually used anywhere.

Reply:
@darktrojan If you are just testing this locally, you can run this [script](https://github.com/mozilla/fxa/blob/1976e6ed79f1235d52cdf3260d5ce56f865420bb/packages/fxa-auth-server/scripts/generate-client-for-ops.js#L12) to create a client. Then add those values to the dev.json file.